mac/autobuild/artifacts/autobuild_mac_2_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# autobuild_2_build: actual build process
+
+if [ ! -z "${1}" ]; then
+	THIS_JAMULUS_PROJECT_PATH="${1}"
+elif [ ! -z "${jamulus_project_path}" ]; then
+	THIS_JAMULUS_PROJECT_PATH="${jamulus_project_path}"
+elif [ ! -z "${GITHUB_WORKSPACE}" ]; then
+	THIS_JAMULUS_PROJECT_PATH="${GITHUB_WORKSPACE}"
+else
+    echo "Please give the path to the repository root as environment variable 'jamulus_project_path' or parameter to this script!"
+    exit 1
+fi
+
+cd ${THIS_JAMULUS_PROJECT_PATH}
```

Comment:
I assume I wrote some of this a few months ago and I didn't know about the quoting issue. Could you please add quotes here?

Reply:
All the bash scripts need checking.